src/main/java/org/intellij/sdk/codesync/files/ConfigFile.java

```diff
@@ -32,9 +31,7 @@ public ConfigFile(String filePath) throws InvalidConfigFileError {
         try {
             this.contentsMap = this.readYml();
         } catch (InvalidYmlFileError e) {
-            CodeSyncYmlFile.removeFileContents(this.getYmlFile());
-            StateUtils.reloadState(StateUtils.getGlobalState().project);
-            CodeSyncLogger.error("Removed contents of the config file after facing invalid yaml error.");
+            CodeSyncLogger.error("[CONFIG_FILE]: Config file facing invalid yaml error.");
```

Comment:
How is this logger function different from a normal logger statement?

Reply:
It is the same, I just changed the message.

Reply:
I meant this method CodeSyncLogger, could you tell me what it is doing?
![image](https://github.com/codesyncapp/codesync-intellij/assets/15974895/9c5d4a75-ae2d-4fc9-a836-e941d5a24b42)